pr2_pick_perception/ItemDescriptor descriptor
string label
